bionic/dockerfile/build/Dockerfile

```diff
@@ -16,5 +16,5 @@ RUN echo "debconf debconf/frontend select noninteractive" | debconf-set-selectio
   apt-get -y $package_args install $packages && \
   rm -rf /var/lib/apt/lists/* /tmp/*
 
-RUN curl -sL -o /usr/local/bin/yj https://github.com/sclevine/yj/releases/latest/download/yj-linux \
+RUN curl -sL -o /usr/local/bin/yj https://github.com/sclevine/yj/releases/latest/download/yj-linux-amd64 \
```

Comment:
I'd suggest `-sSfL` here. The `f` will ensure a non-zero exit in the event of a bad status like 404 and the `S` would cause the `s` to not suppress the output in case of errors.

Kudos to @cezarsa for pointing this to me all the time in my PRs :sweat_smile:

Reply:
That sounds reasonable to me. @paketo-buildpacks/stacks-maintainers would you like a separate PR for this proposed change to the curl flags? A separate PR will make each change easier to reason about and revert if necessary, but they're also both fairly small and clear changes so I'm happy to make it inline.

Reply:
Fine with doing this in the same PR!